packages/vue/src/components/molecules/SfRadio/SfRadio.vue

```diff
@@ -20,6 +20,8 @@
       <!-- @slot Custom checkmark markup (bind 'isChecked' boolean, 'disabled' boolean -->
       <slot name="checkmark" v-bind="{ isChecked, disabled }">
         <div
```

Comment:
IMO we should set it for native input and use CSS to transfer it for our template, I will test some idea tomorrow

Reply:
I figured it out, please check again @pspaczek